apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_components/claim-conditions/claim-conditions-form/hooks.ts

```diff
@@ -208,6 +208,7 @@ function toBigInt(value: string | number | undefined): bigint | undefined {
   if (value === "unlimited") {
     return maxUint256;
   }
+  return BigInt(value);
```

Comment:
this is the actual fix... `toBigInt` was never returning the actual... bigint

Reply:
cc @kien-ngo for your other PR that is related i think?

Reply:
ugrr I accidentally moved that [return BigInt] to another function (for the erc20 fix) - mb!!